### DISCUSSION TITLE
Rescan of photo for face recognition

### DISCUSSION THREAD

**Discussion by DeltaTango69** [immich-app/immich#7750] [Q&A] [answered]
Hello there,

I have some photos where unfortunately no face was recognized. Is there any way to scan them again - perhaps with more precise recognition - i.e. some button with **face recognition with double precision**?

**Comment by BLACK4585** [comment]
Same question here. Is the only way currently running FR for all photos again? And what does the warning popup mean? Will this action delete all persons (with names, birthdates and so on) or just remove their connections to the photos?

**Reply by mertalev** [reply]
It'll delete the people entirely.

**Comment by mertalev** [comment, marked as answer]
Running facial recognition on a particular image is planned, but not implemented yet. I think the only way right now is to delete the image and re-add it with the new settings.

**Reply by MerlinBlatterer** [reply]
I would like to rescan all photos to see if additional faces can be recognized. Is this already available/planned?

**Reply by mertalev** [reply]
There's now a "Refresh" button in the job panel next to Face Detection that will do this. It only makes sense to run it if you changed the face detection threshold, or else the result will be the same.

**Reply by Kusnierz-Maciej** [reply]
I was testing this feature and deleted recoginzed face from one photo.
Rescanning, refhresing on photo and on jobs level didn't bring that face back.

**Reply by aviv926** [reply]
> I was testing this feature and deleted recoginzed face from one photo. Rescanning, refhresing on photo and on jobs level didn't bring that face back.

You can follow this issue here
https://github.com/immich-app/immich/discussions/21822

**Comment by aviv926** [comment]
It is important to know that sometimes you may have set the value MIN RECOGNIZED FACES higher than 3, this means that if the person does not appear and is recognized at least 3 times in the entire Immich library, he will not be displayed under people.

 Therefore, you should pay attention to this because this could be the reason why the face does not appear in the "People" section.

**Comment by agster27** [comment]
I would like this feature as well. I have imported over 100,000 images and every so often I run into one that has not recognized a face or two that is clearly in view.
